.github/workflows/review-code.yaml

```
@@ -0,0 +1,53 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout PR Code
+        uses: actions/checkout@v2
+
+      - name: Fetch All History for Diff
+        run: git fetch --all
+
+      - name: Get PR Diff and File Paths
+        id: diff
+        run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)